css-view-transitions-1/Overview.bs

```diff
@@ -1015,9 +1015,17 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 		:: a {{Promise}}.
 			Initially [=a new promise=] in [=this's=] [=relevant Realm=].
 
+			Note: These promises are immediately created,
+			so rejections will cause {{unhandledrejection}}s unless they're [=mark as handled|handled=],
```

Comment:
Can you clarify this a bit: Which rejections will cause unhandledrejection? Is it that if this promise is rejected before it is handled, then it will cause an unhandledrejection? 

Also "these promises": that's the ready promise and the update callback done promise, but not the finished promise, right?

Reply:
> Can you clarify this a bit: Which rejections will cause unhandledrejection? Is it that if this promise is rejected before it is handled, then it will cause an unhandledrejection?

Pretty much. If a promise is rejected, but not handled before the next task, it becomes an unhandled rejection. I think it's a kinda rough edge of promises tbh https://jakearchibald.com/2023/unhandled-rejections/.

> Also "these promises": that's the ready promise and the update callback done promise, but not the finished promise, right?

Yeah. This note is too vague. I'll fix that.